docs/phdata.rst

```diff
@@ -339,17 +341,17 @@ particular experiment. If not-relevant these fields are omitted.
 identity group
 --------------
 
-The **identity/** group contains info about the specific Photon-HDF5 file:
+The **identity/** group contains information about the specific Photon-HDF5 file:
 
 - **filename**: (string)
 - **full_filename**: (string)
-- **creation_time**: (string) Creation time as "YYYY-MM-DD HH:MM:SS".
+- **creation_time**: (string) Creation time with the following format: "YYYY-MM-DD HH:MM:SS".
 - **software**: (string)
 - **software_version**: (string)
 - **format_name**: (string) This must always be "Photon-HDF5"
 - **format_version**: (string) "0.3"
 - **format_url**: (string) A URL pointing to the Photon-HDF5 documentation.
-
+(**no DOI?**)
 
```

Comment:
DOI would be in provenance section. This is only automatically-generated metadata, created when saving the file.